<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setTime_win</class>
 <widget class="QWidget" name="setTime_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setTime</string>
  </property>
  <widget class="QLabel" name="label_movName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>filename</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_st1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_st2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_st3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_et3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_et2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_et1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cutVideo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_nextVideo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_movName_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>startTime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_movName_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>42</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>endTime</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_crf">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="maxLength">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_movName_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>crf</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_movName_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>resolution</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_resolution">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ori</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720*405</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>960*540</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024*576</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280*720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920*1080</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_movNameP">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>filename</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
